markdown/en/ff-asn.md

```diff
@@ -25,6 +25,32 @@ If you set the ``[filter-style]` attribute properly the `ff-asn-group` acts as d
 </ff-asn>
 ```
 
+## Using native select box
+
+In version 1.2.7 we introduced native select-box support for `ff-asn-group`.
+Additionally `<option>` element does supports data binding when using
+`<ff-asn-group select-box="true">`. You can now supply your own template
+by annotating it with the data-template attribute.
+
+```html
+<ff-asn-group for-group="xxx" select-box="true">
+        ...
+        <div data-container="detailedLinks">
+            <div data-content="detailedLinks"></div>
+        </div>
+
+        <select data-container="hiddenLinks">
+            <option>more ...</option>
+            <!--
+                 This template is optional, if you dont supply an template it defaults to:
```

Comment:
Some more grammar.
`This template is optional. If you don't supply one, it defaults to:`